workers/unity/Packages/com.improbable.gdk.core/Worker/WorkerBase.cs

```diff
@@ -21,7 +21,7 @@ public abstract class WorkerBase : IDisposable
 
         public EntityGameObjectLinker EntityGameObjectLinker { get; private set; }
 
-        public bool UseDynamicId = false;
+        public bool UseDynamicId { get; protected set; }
```

Comment:
What is this used for now?

Reply:
to check whether we need to generate a worker id or not. Look at line 34

Reply:
👍 to ☝️ , also to have the `UnityGameLogic` decide whether it wants to be unique per connection or not (as this is expected from the rest of SpatialOS)